spec/test_apps/app/dumpers/tests_response_dumper.rb

```diff
@@ -10,4 +10,17 @@
     get root_path
     delete url_for(controller: :tests, action: :destroy)
   end
+
+  dump 'multipart_formdata_request_with_file', status_codes: %i[no_content] do
+    mocked_image_file = Rack::Test::UploadedFile.new(
+      StringIO.new('png file data content'),
```

Comment:
Please include some non-ascii / non-UTF-8 binary data in the uploaded file to demonstrate the cases observed during development.

Reply:
Done in 4453fb2fb3da74690f2d5fbc2cc6e07b8dc35bdd; let me know what you think. This includes the PNG magic number, plus a tiny bit more legitimate PNG header hex data from a real file that gives the readout a pinch more substance when viewed in the resulting file. These request_body files are now indeed treated as binaries and I'll update the relevant descriptions to reflect that.